### DISCUSSION TITLE
Application for teacher keeps getting rejected

### DISCUSSION THREAD

**Discussion by jamacis** [community/community#56773] [GitHub Education]
### Hi everyone,

What information is needed for the teacher application to be accepted. I completed profile, application and uploaded a image of my work id. What else is needed.

**Comment by mrepol742** [comment]
**To qualify for teacher benefits, you must:**
- Be an educator, faculty member, or researcher.
- Have a verifiable school-issued email address or upload documents that prove your school affiliation.
- Have a GitHub user account.

https://education.github.com/discount_requests/application

**Comment by jamacis** [comment]
Mr. Jones

I submitted a copy of my CPS work ID, used my school email address to
sign-up and also have a GitHub account. What else is needed?

Thanks
John Macis

**Comment by samus-aran** [comment]
Hi @jamacis 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
